pkg/volume/azure_dd/azure_provision.go

```diff
@@ -241,6 +249,8 @@ func (p *azureDiskProvisioner) Provision(selectedNode *v1.Node, allowedTopologie
 			SizeGB:             requestGiB,
 			Tags:               tags,
 			AvailabilityZone:   selectedAvailabilityZone,
+			DiskIOPSReadWrite:  diskIopsReadWrite,
+			DiskMBpsReadWrite:  diskMbpsReadWrite,
```

Comment:
Those two options only work for UltraSSD disk. Add a pre-check here and fail out if the disk is not UltraSSD.

Reply:
fixed